packages/connection/src/class-connection-error-handler.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * The Jetpack Connection Error Handler class file.
+ *
+ * @package automattic/jetpack-connection
+ */
+
+namespace Automattic\Jetpack\Connection;
+
+/**
+ * The Jetpack Connection Error Handler class is used to handle the errors
+ * generated during the connection process.
+ */
+class Connection_Error_Handler {
```

Comment:
One small nitpick: can we call this class just `Error_Handler`? It's already a Connection package :)

Reply:
Yeah - good point! I'll change it.